docs/modules/ROOT/pages/cli/getting_started_in_cli.adoc

```diff
@@ -0,0 +1,149 @@
+
+== Getting started in the CLI
+
+In addition to the user interface (UI), you can also use the command line interface (CLI) to access and configure {ProductName} to meet your specific needs.
+
+.Prerequisites
+
+To access {ProductName} by using the CLI, you must configure oc commands. Follow the instructions in https://docs.openshift.com/container-platform/4.12/cli_reference/openshift_cli/getting-started-cli.html[Getting started with the OpenShift CLI].
```

Comment:
just confirming that, toward minimalism, you don't want to say "with" instead of "by using".

Reply:
ooh, actually, "oc" is still not in monocode (`oc`)

should i wait and let you create the new PR first?

Reply:
I left it as "by using" since I brought up the issue with the CLI and knowing what preposition to pick -- but I can change this to "with!" 

and oh rats! ill fix it when I do the new PR

Reply:
lol np!! LMK when it's ready and i'll finish my peer rev. :) @chloeredhat